### PR TITLE
feat(webview): cover in loading screen

### PR DESCRIPTION
**Pull request by virtuacoplenny**
third iteration:
no-launch

second iteration (outdated):
black-black

first iteration (outdated):
loading

### CODE REVIEW

**Comment by paweldomas** [comment]
Do you know what is the white screen saying Jitsi Meet Spot ?

File: spot-controller/App.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ export default class App extends React.Component {
         super(props);
 
         this.state = {
+            loading: __DEV__,
```
Comment (paweldomas) [inline spot-controller/App.js]: what does it do ? why set to `__DEV__` ?

Comment (virtuacoplenny) [reply spot-controller/App.js]: Right now only dev does something async (look in local storage). Whereas if it's not dev then it skips looking in local storage. So on dev what happens is the code entry screen displays, find a url or does not, and then displays the webview. But while it's reading from local storage, it shows the code entry screen.

In a follow up commit where I have to store if the app has ever been launched this.state.loading gets started as true, not __DEV__, and will be set to false after async storage has been searched for some flag saying the app has been launched before.

Comment (paweldomas) [reply spot-controller/App.js]: My understanding of the loading screen purpose was to show it during the time when the WebView is loading the page

Comment (virtuacoplenny) [reply spot-controller/App.js]: So you want me to remove hiding the flash of the code entry screen. I don't mind.

Comment (virtuacoplenny) [reply spot-controller/App.js]: I don't know if that's what you want but I went ahead and did it.

**Comment by virtuacoplenny** [comment]
> Do you know what is the white screen saying Jitsi Meet Spot ?

The configured splash screen. I guess I forgot to ask: should the loading screen be white to match the splash screen?

**Comment by virtuacoplenny** [comment]
Looks like launch screen is already removed in latest master. I've rebased.

**Pull request merged**